### PR TITLE
Refresh the README

### PR DESCRIPTION
**Pull request by jfeltesse-mdsol**
- make JSON the preferred tracer
- fix typos
- syntax highlight the code blocks
- make the copywriting better

### CODE REVIEW

**Comment by jfeltesse-mdsol** [comment]
@jcarres-mdsol @ssteeg-mdsol @adriancole @jamescway

File: README.md [hunk 1]
```diff
@@ -60,23 +44,54 @@ gem 'faraday', '~> 0.8'
...
+  # default Faraday stack
+  faraday.request :url_encoded
+  faraday.adapter Faraday.default_adapter
+end
+```
+
+Note that supplying the service name for the destination service is optional;
+the tracing will default to a service name derived from the first section of the destination URL (e.g. 'service.example.com' => 'service').
+
+## Tracers
```
Comment (ssteeg-mdsol) [inline README.md]: Maybe add something at the top explaining that you need to choose between one of the three following

File: README.md [hunk 2]
```diff
@@ -8,47 +8,31 @@ Rack and Faraday integration middlewares for Zipkin tracing.
...
+use ZipkinTracer::RackHandler, config # config is optional
+```
 
- If the configuration does not provide either a JSON, Scribe or Zookeeper server then the middlewares will not
- attempt to send traces although they will still generate proper IDs and pass them to other services.
- Thus, if you only want to generate IDs for instance for logging and do not intent to integrate with Zipkin,
- you can still use this gem. Just do not specify any server :)
+where `Rails.config.zipkin_tracer` or `config` is a hash that can contain the following keys:
 
+* `:service_name` **REQUIRED** - the name of the service being traced. There are two ways to configure this value. Either write the service name in the config file or set the "DOMAIN" environment variable (e.g. 'test-service.example.com' or 'test-service'). The environment variable takes precedence over the config file value.
```
Comment (ssteeg-mdsol) [inline README.md]: Maybe overkill, but I feel like this information would be better in a table

**Comment by ssteeg-mdsol** [comment]
:+1:

File: README.md (hunk 1 above)
Comment (jfeltesse-mdsol) [inline README.md]: There's a note at line 34 stating you have a fourth option: not using any! :santa:

File: README.md (hunk 2 above)
Comment (jfeltesse-mdsol) [inline README.md]: If only making tables in MD wasn't such a PITA and I wasn't so lazy on a Monday morning... :sweat_smile:

Comment (adriancole) [inline README.md]: Lol

**Comment by adriancole** [comment]
:shipit:

**Comment by jcarres-mdsol** [comment]
Obviously a README has not broken the specs but travis is not passing, can your force push some change to make it run again?

**Comment by adriancole** [comment]
no worries. I'll merge

**Pull request merged**